### ISSUE TITLE
Use consistent name for PostgreSQL database across project

### ISSUE THREAD

**Issue by erictapen** [Nix-Security-WG/nix-security-tracker#242]
Found values so far:

- web-security-tracker
- nix-security-tracker

I'd vote for nix-security-tracker, as it's also the repository name.

**Comment by fricklerhandwerk** [comment]
Yes, let's use `nix-security-tracker` everywhere.

**Comment by RaitoBezarius** [comment]
This needs a rename of our staging deployment's DB, I can take care of it.

**Comment by erictapen** [comment]
Yeah that would be great, because the dump has the name of the role all over the place. I used sed in the past to be able to restore a dump to my local dev database.
